requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
+# This file is autogenerated by pip-compile with python 3.10
```

Comment:
Please stick to Python 3.9 for this project, these are the kind of diffs we don't want to see. When in doubt. look at the python version used in `Dockerfile`, as that's what gets built for production.

Otherwise these patches/diffs ping-pong back and forth between developers depending on their Python version used, and we don't want that :smile: